TESTING.md

```diff
@@ -0,0 +1,19 @@
+# Testing Chocolatey
+
+Tests for Chocolatey are written in C# or in PowerShell depending on what type of test is being created.
+
+* NUnit Unit Tests are written in C# and found in the `chocolatey.tests` project. They are individual tests of the various Chocolatey components.
+* NUnit Integration tests are written in C# and found in the `chocolatey.tests.integration` project. They are tests of various Chocolatey components that reach outside of Chocolatey.
+* Pester test are written in PowerShell and found in the `tests` directory of the repository. They test the overall integration of Chocolatey into a larger system.
+
+## Running Tests
+
+### NUnit Tests
+
+To run the NUnit tests get run automtically when you run `./build.bat`, you can also run them without building the entire solution by running `./test.bat`.
+
+### Pester Tests
+
+The Pester tests have been modelled in a way to be testable without installing Chocolatey to your local system. We have made efforts to prevent installing software with the tests, or at least uninstalling any that is installed. That being said, we also provide a `Vagrantfile` with a base configuration that will build the Chocolatey package (inside the VM) and run the tests for you.

```

Comment:
We should add an example of how we'd recommend to run the tests on the local system here as well, I think. (Even if it's just `Invoke-Pester tests` or something.)

Reply:
I've updated the document to indicate you can also use the `./Invoke-Tests.ps1` script, as well as including the parameters you can give it.